Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -218,6 +220,8 @@
     sprite: Objects/Weapons/Guns/Shotguns/sawn_inhands_64x.rsi
   - type: Gun
     fireRate: 3
+  - type: GunSpreadModifier # Sunrise-add #Port WZ #37731
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)